### PR TITLE
First stage

### PR DESCRIPTION
**Pull request by TheReturnOfJan**
https://github.com/leapdao/meta/issues/152

### CODE REVIEW

File: src/commonTest/kotlin/test.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        assertTrue { actualOutput == expectedOutput }
+    }
+
+    @Test
+    fun outputIsSerializedCorrectly() {
+        val output = Output(BigInt.fromString("10"), byteArrayFomHexString("0xb1ccdb544f603af631525ec406245909ad6e1b60"), 0.toUShort())
+        val expectedHexString = "0x000000000000000000000000000000000000000000000000000000000000000a0000b1ccdb544f603af631525ec406245909ad6e1b60"
+        val actualHexString = output.toHexString()
+
+        println(expectedHexString)
```
Comment (MaxStalker) [inline src/commonTest/kotlin/test.kt:23]: Do we want to kep debugging here? :)

File: src/commonTest/kotlin/test.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package leapcore
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class CommonTests {
+
+    @Test
+    fun outputIsParsedCorrectly() {
+        val outputHexString = "0x000000000000000000000000000000000000000000000000000000000000000a0000b1ccdb544f603af631525ec406245909ad6e1b60"
```
Comment (MaxStalker) [inline src/commonTest/kotlin/test.kt:10]: Possible to story it outside as constant as we using it twice? (hwere nad on line 20). Same for first argument passed into Output contructor

**Comment by MaxStalker** [comment]
I would appreciate, if we can jump on a call and you would walk me through the code one more time 🙇‍♂️

File: src/commonTest/kotlin/test.kt (hunk 2 above)
Comment (TheReturnOfJan) [reply src/commonTest/kotlin/test.kt:10]: Good point, but this is already obsolete in newer commits.

**Review by MaxStalker — APPROVED** [review]

**Comment by troggy** [comment]
@TheReturnOfJan can you make `master` to be default branch in the repo please?

**Pull request merged**